runtimes/java-11.0/src/main/java/io/openruntimes/java/RuntimeResponse.java

```diff
@@ -0,0 +1,52 @@
+package io.openruntimes.java;
+
+import com.google.gson.Gson;
+
+import java.util.Map;
+
+public class RuntimeResponse {
+    public String data;
+    public int statusCode;
+
+    static final Gson gson = new Gson();
+
+    public RuntimeResponse() {
+    }
+
+    public RuntimeResponse(String data, int statusCode) {
+        this.data = data;
+        this.statusCode = statusCode;
+    }
+
+    public RuntimeResponse json(Map<String, Object> data) {
+        this.data = gson.toJson(data);
+        this.statusCode = 200;
+        return this;
+    }
+
+    public RuntimeResponse json(Map<String, Object> data, int statusCode) {
+        this.data = gson.toJson(data);
+        this.statusCode = statusCode;
+        return this;
+    }
+
+    public RuntimeResponse send(String data) {
+        this.data = data;
+        this.statusCode = 200;
+        return this;
+    }
+
+    public RuntimeResponse send(String data, int statusCode) {
+        this.data = data;
+        this.statusCode = statusCode;
+        return this;
+    }
+
+    public static RuntimeResponse error(Throwable error) {
+        return new RuntimeResponse("{\"message\": \"" + error.getMessage() + "\"}", 500);
```

Comment:
Same here, we no longer return a JSON string for the error 
https://github.com/open-runtimes/open-runtimes/blob/3261573d3a02e7dbad1868e465d2858a3a183494/runtimes/deno-1.13/server.ts#L45